src/main/resources/TotemCode/TotemSheet.xml

```diff
@@ -763,69 +746,60 @@ initColorPickers : function(){
  $(XWModalContent).addClass("ajaxloader");
- $(XWModalContent).load(attPickerURL,function(){
+ $(XWModalContent).load(attPickerURL,function() {
  $(XWModalContent).removeClass("ajaxloader");
-   //Initialize attachment picker btns
-   $("input[name='action_upload'][type='submit']").click(function(event){
-      //Check input file filed
-      if($("#attachfile").val() == ""){//Empty field
+   // Initialize attachment picker btns
+   $("input[name='action_upload'][type='submit']").click(function(event) {
+      // Check input file filed
+      if ($("#attachfile").val() == "") {// Empty field
            new XWiki.widgets.Notification("$services.localization.render('xe.attachmentSelector.upload.error.noFile')", 'error');
       }
       else
       {
-           //Check allowed extensions
+           // Check allowed extensions
            var allowedExts = $("#attachfile").attr("title");
            allowedExts = (allowedExts != "") ? allowedExts.split(', ') : new Array();
            var fileExt = null;
            var filename = $("#attachfile").val();
            var fpath = filename.replace('\\', '/');
            var pieces = fpath.split('/');
-           if(pieces.length &gt; 0){
+           if (pieces.length &gt; 0) {
                var basename = pieces[pieces.length-1];
                pieces = basename.split('.');
                fileExt = pieces[pieces.length-1].toLowerCase();
            }
-           if(fileExt &amp;&amp; allowedExts.indexOf(fileExt) == -1){
+           if (fileExt &amp;&amp; allowedExts.indexOf(fileExt) == -1) {
                new XWiki.widgets.Notification("$services.localization.render('xe.attachmentSelector.upload.error.badExtension')", 'error');
            }
            else{
-               var uploadURL = $("#uploadAttachment").attr('action');
-               var params = $("#uploadAttachment :not(input[name='xredirect'])").serialize();
-               var xredirect = "$util.encodeURI($xwiki.getURL('XWiki.SuggestDisplayEditItems','view','xpage=plain&amp;outputSyntax=plain&amp;action=ajaxUploadAttachment'))";
-               uploadURL = uploadURL + "?" + params + "&amp;xredirect=" + xredirect;
                new XWiki.widgets.Notification("$services.localization.render('xe.attachmentSelector.upload.inProgress')", 'inprogress');
-               jQuery.ajaxFileUpload({
-                  url:uploadURL,
-	          secureuri:false,
-   	          fileElementId:'attachfile',
-	          success: function (data, status){
-                  },
-                  complete:function(data, status){
-                      //Reload the page after upload
-                      $(".xnotification-inprogress").remove();
-                      $.initAttachmentPicker(editFormURL,attPickerURL,XWModalContent);
-                  },
-                  error: function (data, status, e){
-	          }
-               });
+               require(['jquery-form'],function(jqf) {
```

Comment:
If you do what I suggested you in the previous comment, you would not need this line anymore.